controllers/ramenconfig.go

```diff
@@ -52,6 +52,7 @@ const (
 	drClusterOperatorChannelNameDefault               = "alpha"
 	drClusterOperatorCatalogSourceNameDefault         = "ramen-catalog"
 	drClusterOperatorClusterServiceVersionNameDefault = drClusterOperatorPackageNameDefault + ".v0.0.1"
+	DefaultCephFSCSIDriverName                        = "openshift-storage.cephfs.csi.ceph.com"
```

Comment:
(nit) default should be the upstream [default](https://github.com/ceph/ceph-csi/blob/2a6487cbf5a71e58aa95baf594dc621bda4fbb8e/cmd/cephcsi.go#L46) driver name, which is `cephfs.csi.ceph.com`

Reply:
Interesting - @ShyamsundarR is it possible we'll be running in envs where the installed driver is `cephfs.csi.ceph.com`?  With the current code, the user would need to set the value in the ramenconfig to match the driver - so we were hoping to get the default correct (so by default we use the workaround even without user needing to modify ramenconfig).

Reply:
We would default the config value to the openshift default in the product, and global default can remain the upstream version (ideally). So even if we changed this to the upstream default, we will add a value in the config for downstream builds.